onset_maproom/maproom.py

```diff
@@ -111,11 +112,11 @@ def onset_plots(click_lat_lng, search_start_day, search_start_month, searchDays,
     lat, lng = get_coords(click_lat_lng)
     try:
         precip = rr_mrg.precip.sel(X=lng, Y=lat, method="nearest", tolerance=0.04)
-        isnan = np.isnan(precip)
-        if any(isnan) == True:
+        isnan = np.isnan(precip).sum().sum()
+        if isnan > 0:
```

Comment:
@aaron-kaplan This speeds up the maproom greatly. I also think this could be useful if we decide on some sort of threshold of missing values?

Reply:
whatever!